infrastructure/protocol-upgrade/src/utils.ts

```diff
@@ -46,7 +50,7 @@ export function getCommonUpgradePath(): string {
 
 export function getUpgradePath(environment: string): string {
     const upgradeEnvironment = environment ?? 'localhost';
-    const path = `${getCommonUpgradePath()}/${upgradeEnvironment}`;
+    const path = `${getCommonUpgradePath()}${upgradeEnvironment}`;
```

Comment:
are you sure?

Reply:
getCommonUpgradePath already ends in a "/", I found this when actually using the tool. So yes.